Services/CustomsService.cs

```diff
@@ -0,0 +1,265 @@
+using CustomsController.Model;
+using Microsoft.EntityFrameworkCore;
+
+namespace CustomsController.Services;
+
+/// <inheritdoc/>
+public class CustomsService : ICustomsService
+{
+    private readonly CustomsContext _customContext;
+
+    /// <summary>
+    /// Constructor
+    /// </summary>
+    /// <param name="customsContext">for logic</param>
+    public CustomsService(CustomsContext customsContext)
+    {
+        _customContext = customsContext;
+    }
+
+    /// <inheritdoc/>
+    public Country AddNewCountry(string A2Code, bool isEUCU)
+    {
+        var country = new Country(A2Code, isEUCU);
```

Comment:
you can do the check before creating the country object

Reply:
ok